cli/cmd/plugin/unmanaged-cluster/README.md

```diff
@@ -0,0 +1,402 @@
+# Unmanaged
+
+`unmanaged-cluster` provides single-node, static, Tanzu clusters. It is ideal for local
+workstation or single-node deployments. It is not meant for production workloads
+and does not offer cluster lifecycle capabilities. For needs involving
+cluster lifecycle, use the `tanzu management-cluster` feature.
+
+[unmanaged-video](https://user-images.githubusercontent.com/23109390/140801679-813a9f72-ded9-4453-b46b-75adb61e7ce2.mp4)
+
+This feature is currently
+[under proposal](https://github.com/vmware-tanzu/community-edition/issues/2266) and users should
+expect instability. We appreciate all usage feedback to be added to [the
+issue](https://github.com/vmware-tanzu/community-edition/issues/2266).
+
+## Usage
+
+### Setup
+
+> _**Pre-reqs**: Currently, you must have docker available on the workstation you're
+using `unmanaged-cluster`. This could be in the form of the docker daemon (Linux) or Docker Desktop
+(Mac/Windows)._
+
+!! These instructions assume you have an existing install of the `tanzu` CLI !!
+
+1. Download the (unsigned) binary.
+
+    * [Linux](https://storage.googleapis.com/tce-tanzu-cli-plugins/build-tools/unmanaged/linux/tanzu-unmanaged-cluster)
```

Comment:
These Google storage links appear broken and failing now in the URL linter. When I click the link, I see:

![Screen Shot 2022-01-10 at 3 12 36 PM](https://user-images.githubusercontent.com/23109390/148847373-b19899de-dae9-4b10-9883-dff12cfeed73.png)

Reply:
The binaries appear to still be at:

https://storage.googleapis.com/tce-tanzu-cli-plugins/build-tools/standalone/linux/tanzu-standalone

do we want to build some new ones and put them in the right bucket?

Reply:
> Or should we remove this chunk in the README? We probably don't need a pre-built binary of it if it's in the main release and on the main branch

My vote is to remove it in the README. Totally agreed that the pre-built binary isn't required and since the RFC window has close, folks can either:

1. get it in the v0.10.0 release
2. build from source

thoughts?